packages/list/index.tsx

```diff
@@ -129,7 +129,7 @@ export default class List extends React.Component<ListProps, ListState> {
     }
     this.foundation.setWrapFocus(wrapFocus!);
     this.foundation.setVerticalOrientation(
-      this.props[ARIA_ORIENTATION] === VERTICAL
+      this.props.orientation !== HORIZONTAL
```

Comment:
Please leave a comment here as to why you're not using the affirmative (`=== VERTICAL`)